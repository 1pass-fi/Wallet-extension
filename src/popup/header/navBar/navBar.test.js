import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'


import Navbar from './index'

import { BrowserRouter as Router } from 'react-router-dom'

describe('Test for Navbar component', () => {
  describe('Render without crashing', () => {
    it('renders correctly', () => {
      const { container } = render(<Router><Navbar/></Router>)
      expect(container).toMatchSnapshot()
    })
  })
})
