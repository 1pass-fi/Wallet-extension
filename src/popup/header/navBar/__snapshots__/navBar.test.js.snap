// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Navbar component Render without crashing renders correctly 1`] = `
<div>
  <nav>
    <a
      class="nav-item"
      href="/account/home"
    >
      <div
        class=""
      >
        Accounts
      </div>
    </a>
    <a
      class="nav-item"
      href="/assets"
    >
      <div
        class=""
      >
        Assets
      </div>
    </a>
    <a
      class="nav-item"
      href="/activity"
    >
      <div
        class=""
      >
        Activity
      </div>
    </a>
  </nav>
</div>
`;
